package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "flow": "flow",
     "license-check": "lerna run license-check",
-    "lint": "yarn run lint-css -s; a=$?; yarn run lint-js -s; b=$?; test $a -eq 0 -a $b -eq 0",
+    "lint": "yarn run lint-css; a=$?; yarn run lint-js; b=$?; test $a -eq 0 -a $b -eq 0",
```

Comment:
`run` is also redundant 

"yarn lint-css; a=$?; yarn lint-js; b=$?; test $a -eq 0 -a $b -eq 0"

should also work :)

Reply:
True. I thought that it would maybe influence compatibility. :)